cudaaligner/src/batched_device_matrices.cuh

```diff
@@ -23,9 +23,9 @@ namespace claragenomics
 namespace cudaaligner
 {
 
-__device__ inline bool error(int t)
+__device__ inline bool error(int x, int y)
```

Comment:
to be consistent, let's change the `int` to `int32_t`?

Reply:
good point. fixed.